common/include/pcl/common/impl/io.hpp

```diff
@@ -460,5 +463,6 @@ pcl::copyPointCloud (const pcl::PointCloud<PointT> &cloud_in, pcl::PointCloud<Po
   }
 }
 
-#endif // PCL_IO_IMPL_IO_H_
+} // namespace pcl
 
+#endif // PCL_IO_IMPL_IO_H_
```

Comment:
Replace this `#if` with a `#pragma once`

Reply:
I was thinking different PR, I'm touching a lot of things already, I feel.